src/test/scala/tech/cryptonomic/conseil/tezos/DatabaseConversionsTest.scala

```diff
@@ -20,7 +22,7 @@ class DatabaseConversionsTest
     val groupHash = OperationHash("operationhash")
 
     //keep level 1, dropping the genesis block
-    val block = generateSingleBlock(atLevel = 1, atTime = testReferenceDateTime)
+    val (genesis :: block :: Nil) = generateBlocks(toLevel = 1, startAt = testReferenceDateTime)
```

Comment:
What happens when ``generateBlocks`` returns one-element collection?

Reply:
when it happens we need to also rewrite the pattern matching val binding

Reply:
To be more precise, the test will fail as soon as you change the parameters that drives the number of  generated blocks.
The code will still compile, though, which is a shame, I agree.

Reply:
Ahhh, ok. I didn't realize that this is a test and I fought that we have no control over what comes from Tezos.